docs/develop/specify-office-hosts-and-api-requirements.md

```diff
@@ -86,99 +62,139 @@ The `Hosts` element can contain one or more `Host` elements. The `Host` element
+- The **Methods** element can contain one or more [Method](../reference/manifest/method.md) elements. You can't use the **Methods** element with Outlook add-ins.
+- The **Method** element specifies an individual method that the Office application must support to make the add-in installable. The `Name` attribute is required and specifies the name of the method qualified with its parent object.
 
-- The `Sets` element can contain one or more `Set` elements. `DefaultMinVersion` specifies the default `MinVersion` value of all child `Set` elements.
+## Design for alternate experiences
 
-- The `Set` element specifies requirement sets that the Office application must support to run the add-in. The `Name` attribute specifies the name of the requirement set. The `MinVersion` specifies the minimum version of the requirement set. `MinVersion` overrides the value of `DefaultMinVersion` For more information about requirement sets and requirement set versions that your API members belong to, see [Office Add-in requirement sets](../reference/requirement-sets/office-add-in-requirement-sets.md).
+The extensibility features that the Office add-in platform provides can be usefully divided into three kinds:
 
-- The `Methods` element can contain one or more `Method` elements. You can't use the `Methods` element with Outlook add-ins.
+- Extensibility features that are available immediately after the add-in is installed. You can make use of this kind of feature by configuring a [VersionOverrides](../reference/manifest/versionoverrides.md) element in the manifest. An example of this kind of feature is [Add-in Commands](../design/add-in-commands.md), which are custom ribbon buttons and menus.
+- Extensibility features that are available only when the add-in is running and that are implemented with Office.js JavaScript APIs; for example, [Dialog Boxes](../design/dialog-boxes.md).
+- Extensibility features that are available only at runtime but are implemented with a combination of Office.js JavaScript and configuration in a **VersionOverrides** element. Examples of these are [Excel custom functions](../excel/custom-functions-overview.md), [single sign-on](sso-in-office-add-ins.md), and [custom contextual tabs](../design/contextual-tabs.md).
 
-- The `Method` element specifies an individual method that must be supported in the Office application where your add-in runs. The `Name` attribute is required and specifies the name of the method qualified with its parent object.
+If your add-in uses a specific extensibility feature for some of its functionality but has other useful functionality that doesn't require the extensibility feature, you should design the add-in so that it is installable on platform and Office version combinations that don't support the extensibility feature. It can provide a valuable, albeit diminished, experience on those combinations. 
```

Comment:
```suggestion
If your add-in uses a specific extensibility feature for some of its functionality but has other useful functionality that doesn't require the extensibility feature, you should design the add-in so that it's installable on platform and Office version combinations that don't support the extensibility feature. It can provide a valuable, albeit diminished, experience on those combinations. 
```